docs/release-notes/1.8.0.asciidoc

```diff
@@ -0,0 +1,43 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.8.0]]
+== {n} version 1.8.0
+
+[[breaking-1.8.0]]
+[float]
+=== Breaking changes
+
+* Remove non-functional pre-stop hook parts {pull}4801[#4801] (issue: {issue}4698[#4698])
+
+The operator does not rely on the `$PRE_STOP_MAX_WAIT_SECONDS` environment variable to control <<{p}-prestop,Elasticsearch Pods PreStop hook>> anymore. Use `$PRE_STOP_ADDITIONAL_WAIT_SECONDS` instead (defaults to 50 seconds).
+
+* Note the next minor ECK version (1.9.0) will drop support for Openshift 3.11.
```

Comment:
Added that as breaking change to make sure people notice, but it's also explained in the release highlights. So maybe I should remove it from here?

Reply:
I think we should leave it here, I'd rather duplicate that info than risk someone getting surprised.